### PR TITLE
Pioneer v1 update

### PR DESCRIPTION
**Pull request by DzhideX**
Changes:
- Remove non-working modules
- Add new opening text and Pioneer v2 section to home page

### CODE REVIEW

File: pioneer/packages/apps-config/src/settings/endpoints.ts [hunk 1]
```diff
@@ -31,30 +31,35 @@ function createLive (t: TFunction): LinkOption[] {
...
     {
       info: 'joystream',
-      text: t<string>('rpc.joystream-5', 'Joystream (Singapore)', { ns: 'apps-config' }),
-      value: 'wss://testnet-rpc-2-singapore.joystream.org'
+      text: t<string>('rpc.joystream-2', 'Joystream (Old Giza Network)', { ns: 'apps-config' }),
+      value: 'wss://rome-rpc-endpoint.joystream.org:9944'
     }
     // {
+    //   info: 'joystream',
+    //   text: t<string>('rpc.joystream-2', 'Joystream (JoystreamStats.Live)', { ns: 'apps-config' }),
```
Comment (traumschule) [inline pioneer/packages/apps-config/src/settings/endpoints.ts:44]: This load balanced endpoint will stay available and https://pioneer.joystreamstats.live/ is going to be updated too:
```
    {
      info: 'joystream',
      text: t<string>('jsstats.olympia', 'Joystream (JoystreamStats.Live)', { ns: 'apps-config' }),
      value: 'wss://olympia.joystreamstats.live:9945'
    },
```

File: pioneer/packages/joy-media/src/index.tsx [hunk 2]
```diff
@@ -62,14 +75,51 @@ const App: React.FC<Props> = () => {
       <Header>
         <h1>Hello there!</h1>
         <p>
-          We have now upgraded to the Sumer chain.
+        We have just upgraded to our new Olympia Chain. That means our clunky user interface has been replaced,
+        and all the action will happen on <a target='_blank' rel='noopener noreferrer' href='https://www.dao.joystream.org'>dao.joystream.org</a>.
```
Comment (bwhm) [inline pioneer/packages/joy-media/src/index.tsx]:
```suggestion
        and all the action will happen on <a target='_blank' rel='noopener noreferrer' href='https://dao.joystream.org'>dao.joystream.org</a>.
```

File: pioneer/packages/joy-media/src/index.tsx [hunk 3]
```diff
@@ -62,14 +75,51 @@ const App: React.FC<Props> = () => {
...
+            <li>
+              As the new pioneer doesn&apos;t store keys, you need to copy all your keys over to the{' '}
+              <a target='_blank' rel='noopener noreferrer' href='https://polkadot.js.org/extension/'>polkadot-js extension</a>.
+            </li>
+            <li>If you want to be a Validator on Olympia, you will have to do it here for now!</li>
+          </ol>
+        </ListParagraph>
         <p>
-          Pioneer <b>no longer supports</b> media uploads and consumption.
+          More information can be found <a target='_blank' rel='noopener noreferrer' href='https://blog.joystream.org/olympia-released/'>here</a>!
```
Comment (bwhm) [inline pioneer/packages/joy-media/src/index.tsx]: This needs to be scrapped, as the blog post won't go live yet...

**Review by bwhm — CHANGES_REQUESTED** [review]
Just two small fixes needed!

**Comment by bwhm** [comment]
Looks good now!

**Pull request merged**